lib/forked-actor-parent.js

```diff
@@ -115,12 +115,19 @@ class ForkedActorParent extends ForkedActor {
 
       // Handle debugging: increment debugger port for child process.
       let execArgv = _.map(process.execArgv, arg => {
-        let match = arg.match(/^--debug-brk=(\d+)/);
+        let match = arg.match(
+          /^(--inspect(?:-brk)?|--debug(?:-brk)?)(?:=([\d\.]+)(?::(\d+))?)?/
+        );
 
         if (match) {
-          let debugPort = parseInt(match[1]);
+          const [, cmd, host, port] = match;
+          let debugPort = parseInt(port);
 
-          return '--debug-brk=' + (debugPort + this.debugPortCounter++);
+          let incPortCmd = cmd;
+          incPortCmd += host ? '=' + host : '';
+          incPortCmd += debugPort ? ':' + (debugPort + this.config.n + 1) : '';
```

Comment:
I think we need a more indicative name for actor number parameter. Let's call it `actorNumber` instead of `n`.

Reply:
Had renamed.